package.json

```diff
@@ -42,9 +44,11 @@
   ],
   "license": "MIT",
   "devDependencies": {
+    "concurrently": "^8.2.0",
     "jest": "^29.6.2",
     "jest-environment-jsdom": "^29.6.2",
     "jest-junit": "^16.0.0",
+    "lite-server": "^2.6.1",
```

Comment:
Looks like `lite-server` is largely unmaintained, the last release has been 3 years ago. Looks like it also is being flagged as a dependency with known vulnerabilities.

Looking around, perhaps [`browser-sync`](https://www.npmjs.com/package/browser-sync) could be an alternative? It seems to be actively maintained, and it certainly [is a lot more popular](https://npmtrends.com/browser-sync-vs-lite-server).

Reply:
Yeah, lite-server wraps browsersync, so i will look into using native Browsersync instead

Reply:
Ha! Didn't even notice it was a dependency of `lite-server`. Yeah, then it totally makes sense to use it directly if it has the right feature set.